diff --git a/node_modules/@types/react/index.d.ts b/node_modules/@types/react/index.d.ts
index 69e0fea..1aa715d 100644
--- a/node_modules/@types/react/index.d.ts
+++ b/node_modules/@types/react/index.d.ts
@@ -1743,7 +1743,7 @@ declare namespace React {
      * @version 16.8.0
      * @see {@link https://react.dev/reference/react/useRef}
      */
-    function useRef<T>(initialValue: T | undefined): RefObject<T | undefined>;
+    function useRef<T>(initialValue?: T | undefined): RefObject<T | undefined>;
     /**
      * The signature is identical to `useEffect`, but it fires synchronously after all DOM mutations.
      * Use this to read layout from the DOM and synchronously re-render. Updates scheduled inside
