--- a/node_modules/react-native-linear-gradient/index.android.js
+++ b/node_modules/react-native-linear-gradient/index.android.js
@@ -3,7 +3,7 @@
  * @flow
  */
 import React, { Component, createRef } from 'react';
-import { processColor, StyleSheet, View } from 'react-native';
+import { StyleSheet, View } from 'react-native';

 import NativeLinearGradient, { type Props } from './src';

@@ -85,7 +85,7 @@ export default class LinearGradient extends Component<Props> {
       <View ref={this.gradientRef} {...otherProps} style={style}>
         <NativeLinearGradient
           style={{position: 'absolute', top: 0, left: 0, bottom: 0, right: 0}}
-          colors={colors.map(processColor)}
+          colors={colors}
           startPoint={convertPoint('start', start)}
           endPoint={convertPoint('end', end)}
           locations={locations ? locations.slice(0, colors.length) : null}
diff --git a/node_modules/react-native-linear-gradient/index.ios.js b/node_modules/react-native-linear-gradient/index.ios.js
index 489f214..4d46060 100644
--- a/node_modules/react-native-linear-gradient/index.ios.js
+++ b/node_modules/react-native-linear-gradient/index.ios.js
@@ -3,7 +3,6 @@
  * @flow
  */
 import React, { Component, createRef } from 'react';
-import { processColor } from 'react-native';

 import NativeLinearGradient, { type Props } from './src';

@@ -56,7 +55,7 @@ export default class LinearGradient extends Component<Props> {
         {...otherProps}
         startPoint={convertPoint('start', start)}
         endPoint={convertPoint('end', end)}
-        colors={colors.map(processColor)}
+        colors={colors}
         locations={locations ? locations.slice(0, colors.length) : null}
         useAngle={useAngle}
         angleCenter={convertPoint('angleCenter', angleCenter)}
